data/json/monsters/nether.json

```
@@ -1329,7 +1329,7 @@
     "id": "mon_hallucinator",
     "type": "MONSTER",
     "name": { "str": "person" },
-    "description": "A common human, that looks exactly like human do.  You can see it.  It can see you.  You know it knows you can see it.  It knows you know you can see him.  It moves towards you.",
+    "description": "A common human, that looks exactly like humans do.  You can see it.  It can see you.  You know it knows you can see it.  It knows you know you can see him.  It moves towards you.",
```

```suggestion
    "description": "A common human, that looks exactly like humans do.  You can see it.  It can see you.  You know it knows you can see it.  It knows you know you can see it.  It moves towards you.",
```

It's been bugging me for a long while with this description. If you can fix that up while you're at it I'd appreciate it.